src/mono/mono/metadata/profiler-events.h

```diff
@@ -32,8 +32,6 @@
 #endif
 
 MONO_PROFILER_EVENT_0(runtime_initialized, RuntimeInitialized)
-MONO_PROFILER_EVENT_0(runtime_shutdown_begin, RuntimeShutdownBegin)
```

Comment:
Isn't this part of the embedding API ?

Reply:
I thought that one isn't public, but if it is maybe I need to undo that; will check.

Reply:
I think profiler events have API guarantees